third_party/libpg_query/src/pg_query_parse.c

```diff
@@ -9,6 +9,8 @@
 #include <unistd.h>
 #include <fcntl.h>
 
+#define PARSE_STRING 1
```

Comment:
This is the control flag for enabling debug output for postgres parsetree. The control is now added in the same file on line 78